docs/tutorial.md

```diff
@@ -41,22 +43,22 @@ After the connection has been established, RIG sends out a [CloudEvent](https://
 
 Please take note of the `connection_token` in the CloudEvent's `data` field - you need it in the next step.
 
-## 3. Subscribe to a topic
+## 3. Subscribe to a topic [Frontend]
 
 With the connection established, you can create _subscriptions_ - that is, you can tell RIG which events your app is interested in. RIG needs to know which connection you are referring to, so you need to use the connection token you have noted down in the last step:
 
 ```bash
 $ CONN_TOKEN="g2dkAA1ub25vZGVAbm9ob3N0AAACrAAAAAAA"
-$ SUBSCRIPTIONS='{"subscriptions":[{"eventType":"greeting"}]}'
+$ SUBSCRIPTIONS='{"subscriptions":[{"eventType":"chatroom_channel_messages"}]}'
```

Comment:
`chatroom_channel_messages` => `chatroom_message`? The event type is often singular and the `channel` in there doesn't seem to add any meaning.